frontend/src/pages/Login.jsx

```diff
@@ -0,0 +1,9 @@
+const Login = () => {
+  return(
+    <div>
+      <p>Login Page</p>
+    </div>
+  )
+}
+
+export default Login
```

Comment:
Not sure if anyone has strong preferences for default vs named exports for our main pages?

Reply:
all good, we can leave the pages as a default export